crates/lib3h/tests/utils/mod.rs

```diff
@@ -9,8 +9,15 @@ macro_rules! assert_process_success {
         assert!(did_work);
         assert_eq!(srv_msg_list.len(), 1);
         let msg_1 = &srv_msg_list[0];
-        one_let!(Lib3hServerProtocol::SuccessResult(response) = msg_1 {
+        if let Lib3hServerProtocol::SuccessResult(response) = msg_1 {
             assert_eq!(response.request_id, $req);
-        });
+        } else {
+            if let Lib3hServerProtocol::FailureResult(response) = msg_1 {
```

Comment:
since there's multiple `if let`s, should this be a `match` insteadL

Reply:
good point. I'll change that.